### ISSUE TITLE
Select randomly experimental systems and tasks

### ISSUE THREAD

**Issue by hvaldecantos** [hvaldecantos/pcer#19]
Create a RandomSelector class to randomly select experimental systems or tasks:
- It should read the session of the current participant to select randomly the next system or task.
- It should read the `.yml` files in the resource directory to determine the remaining system or tasks.
- It should write the session of the current participant when a system or task is selected

**Comment by hvaldecantos** [comment]
An example of a possible document in the session db:
```
"1": {
            "group": "DCI",
            "id": "10",
            "warmup": True
            "pretest": ["t", "f", "dk", "dk", "t", "t", "t", "f", "dk", "dk", "t"],
            "system": [
            {"library": {"t1": ["t", "f", "f"],
                         "t3": ["t", "f", "f"],
                         "t4": ["t", "f", "f"],
                         "t2": ["t", "f", "f"]},
             "spell" : {"t2" : ["t", "t", "dk"]}}
             ],
        },
```
We might need start and end timestamps for each task and system.

**Comment by hvaldecantos** [comment]
I have created a class Resource responsible of managing the resource (systems, tasks, tests)
See commit sha 72718c8bfe610e4f8f4f3af9333322ac08269d93 in branch `resource-issue#19`

**Comment by hvaldecantos** [comment]
The random selection is made in the Experiment object
https://github.com/hvaldecantos/pcer/blob/1fe195fddf949ca161cb61a6c09a199e5866a1a5/pcer/experiment.py#L59-L66

**Comment by hvaldecantos** [comment]
I changed the DB to:

```
{
    "experiment": {
        "1": {
            "trials": [
                {
                    "timestamp": "2019-07-30 13:25:07.282841", 
                    "finished": false, 
                    "tasks": [
                        {
                            "timestamp": 1234723, 
                            "finished": true, 
                            "answers": { }, 
                            "task_id": "objsinteracts"
                        }, 
                     ], 
                    "system_id": "clock", 
                    "warmup": true
                }
            ], 
            "group": "dci", 
            "id": "123", 
            "warmup_finished": false, 
            "experiment_finished": false
        }
    }
}
```
Pull request #23

**Comment by KaleChinmay** [comment]
Merged pull request #29
